source/common/http/http1/codec_impl.h

```diff
@@ -28,6 +28,9 @@ namespace Envoy {
 namespace Http {
 namespace Http1 {
 
+// The default limit of 80 KiB is the vanilla http_parser behaviour.
+constexpr uint32_t MAX_RESPONSE_HEADERS_KB = 80;
```

Comment:
Not your code, `inline` is  needed for constant in header file to ensure ODR

Reply:
done